crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -396,13 +396,19 @@ where
             info!("Loaded import data from {:?}", path.display());
             contents
         } else {
+            eprintln!("Attempting to import data from stdin ... (Ctrl-D to finish)");
             let mut buffer = String::new();
             let mut stdin = io::stdin();
             let bytes = stdin.read_to_string(&mut buffer)?;
             info!("Loaded {bytes} bytes of import data from stdin");
             buffer
         };
 
+        if contents.is_empty() {
```

Comment:
Maybe could also `trim()` in case was just a blank line or whatever.

Reply:
Nice idea, thanks.